src/metagen/src/client_ts/static/mod.ts

```
@@ -20,10 +20,18 @@ function _selectionToNodeSet(
 
     const { argumentTypes, subNodes, variants, inputFiles } = metaFn();
 
+    const nodeInstances = nodeSelection instanceof Alias
+      ? nodeSelection.aliases()
+      : { [nodeName]: nodeSelection };
     const nodeInstances = nodeSelection instanceof Alias
```

Biome (1.9.4)

[error] Shouldn't redeclare 'nodeInstances'. Consider to delete it or rename it.

'nodeInstances' is defined here:


(lint/suspicious/noRedeclare)